reason-react-native/src/components/ScrollView.md

```diff
@@ -4,6 +4,41 @@ title: ScrollView
 wip: true
 ---
 
+## Methods
+
+### `scrollTo`
+
+```reason
+[@react.component]
+let make = () => {
+  let scrollViewRef = React.useRef(Js.Nullable.null);
+  <ScrollView ref=scrollViewRef>
+    <TouchableOpacity
+      onPress={_ => {
+        scrollViewRef
+        ->React.Ref.current
+        ->Js.Nullable.toOption
+        ->Option.map(scrollView =>
```

Comment:
I am not sure I like the usage of `Option.map` here. Unfortunately Belt does not have `Option.apply`. So maybe just use a `switch`? (Speaking of Belt - you should actually it open it.)